webpack/scenes/ContentViews/Table/tableDataGenerator.js

```diff
@@ -1,137 +1,86 @@
-    {
-      title: <IconWithCount Icon={ScreenIcon} count={environments.length} title={`environments-icon-${id}`} />,
-      props: { isOpen: false, ariaControls: `cv-environments-expansion-${id}` },
-    },
+    { title: lastPublished ? <LongDateTime date={lastPublished} showRelativeTimeTooltip /> : <EmptyContent emptyText="Not yet published" /> },
+    { title: <LastSync lastSync={lastTask} lastSyncWords={lastSyncWords} emptyMessage="N/A" /> },
     {
-      title: <IconWithCount Icon={ContainerNodeIcon} count={versions.length} title={`versions-icon-${id}`} />,
-      props: { isOpen: false, ariaControls: `cv-versions-expansion-${id}` },
+      title: latestVersion ? <ContentViewVersionCell {...{
+ id, latestVersion, latestVersionId, latestVersionEnvironments,
+}}
+      /> : <EmptyContent emptyText="Not yet published" />,
     },
   ];
-  if (openColumn) row[openColumn].props.isOpen = true;
   return row;
 };
 
-const buildDetailDropdowns = (id, rowIndex, openColumn) => {
-  const cvId = { cvId: id };
-  const expansionProps = { ...cvId, className: 'pf-m-no-padding' };
-  const containerProps = column => ({ ...cvId, isOpen: openColumn === column });
-  const offsetColumn = 3; // index of first expandable column
+const buildExpandableRows = (contentViews) => {
+  const rows = [];
+  let cvCount = 0;
 
-  let detailDropdowns = [
-    {
-      compoundParent: offsetColumn,
-      cells: [
-        {
-          title: (
-            <DetailsContainer {...containerProps(offsetColumn + 1)}>
-              <DetailsExpansion {...expansionProps} />
-            </DetailsContainer>),
-          props: { colSpan: 6 },
-        },
-      ],
-    },
-    {
-      compoundParent: offsetColumn + 1,
+  contentViews.forEach((contentView) => {
+    const {
+      id, name, description, activation_keys: activationKeys, hosts,
+    } = contentView;
+    const cells = buildRow(contentView);
+    const cellParent = {
+      cvId: id, cvName: name, isOpen: false, cells,
+    };
+    rows.push(cellParent);
+    const cellChild = {
+      parent: cvCount,
       cells: [
         {
-          title: (
-            <DetailsContainer {...containerProps(offsetColumn + 2)}>
-              <EnvironmentsExpansion {...expansionProps} />
-            </DetailsContainer>),
-          props: { colSpan: 6 },
+          title: <DetailsExpansion cvId={id} {...{ activationKeys, hosts }} />,
+          props: {
+            colSpan: 3,
+          },
         },
-      ],
-    },
-    {
-      compoundParent: offsetColumn + 2,
-      cells: [
         {
-          title: (
-            <DetailsContainer {...containerProps(offsetColumn + 3)}>
-              <VersionsExpansion {...expansionProps} />
-            </DetailsContainer>),
-          props: { colSpan: 6 },
+          title: description ? <>{__('Description: ')}{__(description)}</> : <EmptyContent emptyText="No Description" />,
```

Comment:
```suggestion
          title: description ? <>{__('Description: ')}{__(description)}</> : <EmptyContent emptyText="No description" />,
```
also, wasn't `emptyText` renamed?

Reply:
Ya..that's in the other PR..Will rebase this once that is merged..